src/CommunityToolkit.Aspire.Hosting.Deno/DenoPackageInstaller.cs

```diff
@@ -0,0 +1,121 @@
+using Aspire.Hosting.ApplicationModel;
+using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace CommunityToolkit.Aspire.Hosting.Deno;
+
+/// <summary>
+/// Represents a Deno package installer.
+/// </summary>
+/// <param name="packageManager">The package manager to use.</param>
+/// <param name="loggerService">The logger service to use.</param>
+/// <param name="notificationService">The notification service to use.</param>
+internal class DenoPackageInstaller(string packageManager, string installCommand, string lockfile, ResourceLoggerService loggerService, ResourceNotificationService notificationService)
```

Comment:
We could simplify this by inlining `packageManager, `installCommand` and `lockfile` since they are fixed values in the Deno world.

Reply:
Should be done in latest commit.